hw/top_earlgrey/syn/chip_earlgrey_asic.sdc

```diff
@@ -50,35 +50,71 @@ set USB_CLK_PIN u_ast/clk_src_usb_o
 # target is 48MHz, overconstrain by 5%
 set USB_TCK_TARGET_PERIOD 20.8
 set USB_TCK_PERIOD [expr $USB_TCK_TARGET_PERIOD*$CLK_PERIOD_FACTOR]
-#set_ideal_network [get_pins ${USB_CLK_PIN}]
-
+# USB clock uncertainty needs to be within 2500ppm
+set USB_CLOCK_UNCERTAINTY [expr $USB_TCK_PERIOD * .0025]
```

Comment:
But is it when the clock is not SOF-calibrated? Will we potentially have trouble capturing SOF if this doesn't reflect merely the trimmed values?

Reply:
Ah, I see, we have an extra 5% from ${CLK_PERIOD_FACTOR}. Withdrawn! :smile: